src/DynamoCore/Graph/Workspaces/UndoRedo.cs

```diff
@@ -288,19 +289,20 @@ internal void RecordAndDeleteModels(List<ModelBase> models)
 
                         RemoveAndDisposeNode(node);
                     }
-                    else if (model is ConnectorModel connector)
+                    else if (model is ConnectorModel conn)
```

Comment:
why are we changing the variable name here?

Reply:
The whole snippet here is identical to the one used in the record and deletion of Nodes right above. The two code parts are doing the exact same thing. There shouldn't be a difference between them. I aligned them to better highlight this (we call it `connector` above, and `conn` below - I don't see a reason why)